eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,30 +1,12 @@
 resources:
   repositories:
-  - repository: azure-sdk-build-tools
-    type: git
-    name: internal/azure-sdk-build-tools
-    ref: refs/heads/feature/engsys-ios-barebones
+  - repository: azure-sdk-tools
```

Comment:
Do we still need this repository reference? If so what are we using it for?

cc @chidozieononiwu